### ISSUE TITLE
Please archive the repository

### ISSUE THREAD

**Issue by drm** [mailhog/MailHog#466]
Hi, since this project was proclaimed dead, could you please archive the repository?

**Comment by skyscooby** [comment]
Are you aware if anyone is maintaining a fork? The PR's / Issues might be of value if anyone ever takes it forward..

**Comment by drm** [comment]
Hm, not sure, fair enough. Maybe just add a notice to the readme, then :)

**Comment by SalahAdDin** [comment]
Looking for an alternative!

**Comment by lroehrs** [comment]
Mailpit is our replacement:

https://github.com/axllent/mailpit

**Comment by SalahAdDin** [comment]
> Mailpit is our replacement:
> 
> https://github.com/axllent/mailpit

is it mainly active?

**Comment by oz** [comment]
> > Mailpit is our replacement:
> > https://github.com/axllent/mailpit
> 
> is it mainly active?

@SalahAdDin yes, MailPit is active for now, it's a drop-in replacement except it does not store emails in MailDir format, but relies on sqlite for storage.

**Comment by drm** [comment]
I've been using mailpit and I can definitely recommend it :)

**Comment by skyscooby** [comment]
Yes overall it's significantly better.. besides losing the ability to `grep -r` and `cat` through a maildir.. which was really nice for some simple test cases that now require using an API.. seems like the days of 'simple unix' are slowly dying out..
